### PR TITLE
Wasp reference docs

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added the workflow to pull the wasp references.

- [x] Ignore folders if no refs where generated
- [x] Compare workflow with SDK
- [ ] Change title of Pages (Contract name)

## Links to any relevant issues

Fixes https://github.com/iotaledger/devx/issues/335

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [ ] I have performed a self-review of my own changes
- [ ] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Dr-Electron** [comment]
Related to this I created an issue to use act in the future. I hate the current setup 😅

**Comment by Dr-Electron** [comment]
Sir. When read my todo list 😞

File: docs/build/isc/v1.0.0-rc.6/docs/reference/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+magic-contract/*
+!magic-contract/introduction.md
+iscutils
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/docs/reference/.gitignore:3]:
```suggestion
iscutils

```

File: scripts/get_all_references.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+./get_sdk_references.sh
+./get_wasp_references.sh
```
Comment (Dr-Electron) [inline scripts/get_all_references.sh:2]:
```suggestion
./get_wasp_references.sh

```

**Review by lucas-tortora — APPROVED** [review]
We still have the double headers issue, but that should be sorted in the script that generates and uploads the docs to the bucket.

**Pull request merged**